src/qt/bitcoingui.cpp

```diff
@@ -370,13 +370,12 @@ void BitcoinGUI::createActions()
         connect(openAction, &QAction::triggered, this, &BitcoinGUI::openClicked);
         connect(m_open_wallet_action->menu(), &QMenu::aboutToShow, [this] {
             m_open_wallet_action->menu()->clear();
-            std::vector<std::string> available_wallets = m_wallet_controller->getWalletsAvailableToOpen();
-            std::vector<std::string> wallets = m_node.listWalletDir();
-            for (const auto& path : wallets) {
+            for (const std::pair<std::string, bool>& i : m_wallet_controller->listWalletDir()) {
```

Comment:
```
qt/bitcoingui.cpp:373:54: warning: loop variable 'i' has type 'const std::pair<std::string, bool> &' (aka 'const pair<basic_string<char, char_traits<char>, allocator<char> >, bool> &') but is initialized with type 'std::__1::__map_iterator<std::__1::__tree_iterator<std::__1::__value_type<std::__1::basic_string<char>, bool>, std::__1::__tree_node<std::__1::__value_type<std::__1::basic_string<char>, bool>, void *> *, long> >::value_type' (aka 'pair<const std::__1::basic_string<char>, bool>') resulting in a copy [-Wrange-loop-analysis]
            for (const std::pair<std::string, bool>& i : m_wallet_controller->listWalletDir()) {
                                                     ^
qt/bitcoingui.cpp:373:18: note: use non-reference type 'std::pair<std::string, bool>' (aka 'pair<basic_string<char, char_traits<char>, allocator<char> >, bool>') to keep the copy or type 'const std::__1::__map_iterator<std::__1::__tree_iterator<std::__1::__value_type<std::__1::basic_string<char>, bool>, std::__1::__tree_node<std::__1::__value_type<std::__1::basic_string<char>, bool>, void *> *, long> >::value_type &' (aka 'const pair<const std::__1::basic_string<char>, bool> &') to prevent copying
            for (const std::pair<std::string, bool>& i : m_wallet_controller->listWalletDir()) {
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
```

Reply:
Thanks, fixed.